resources/js/menus/LeftMenu.vue

```diff
@@ -220,22 +239,29 @@ function loadMenu() {
 				},
 			],
 		},
-
 		{
-			label: "lychee.ABOUT_LYCHEE",
-			icon: "info",
-			access: true,
-			command: () => (openLycheeAbout.value = true),
+			label: "Lychee",
+			items: [
+				{
+					label: "About",
```

Comment:
Is `lychee.ABOUT_LYCHEE` no good?

Reply:
About Lychee is returning "About Lychee" and in that specific case it would feel weird to have a sub section "Lychee" and then just inside it "About Lychee" while "About" just get the job done.
But yeah, that is a thing we have to fix later.
Basically redoing the localization of the front-end...

I am really not looking forward to that one. :sweat_smile: